variables.tf

```diff
@@ -92,6 +92,8 @@ variable "master" {
         instance_type = "m1.xlarge"
         # rhcos image id
         image_id      = "468863e6-4b33-4e8b-b2c5-c9ef9e6eedf4"
+        data_volume_size = "100" #Volume size to attach to the master nodes.
+        data_volume_count = "0" #Count of volumes attached to the master nodes.
```

Comment:
Should not be mandatory. Folks using existing var file will start getting errors. Please make them optional with default value in lookup() as 0.